types/office-js/index.d.ts

```diff
@@ -1607,7 +1608,7 @@ declare namespace Office {
      *   <tr><td><strong> Word       </strong></td><td> Y                          </td><td>                            </td><td> Y               </td></tr>
      *  </table>
      */
-    function select(expression: string, callback?: (result: AsyncResult) => void): Binding;
+    function select(expression: string, callback?: (result: AsyncResult<any>) => void): Binding;
```

Comment:
Since it's not anticipated to ever return a value, it should be `AsyncResult<void>` instead